extension/chrome/settings/inbox/inbox.ts

```diff
@@ -454,6 +460,17 @@ Catch.try(async () => {
     }));
   };
 
+  const menuAcctHtml = (email: string, picture = '/img/svgs/profile-icon.svg') => {
```

Comment:
Instead of copy/pasting, please put it into a common place (with a better name). I would put it into `js/common/settings.ts` as a class method like the rest.